regression/goto-cc-goto-analyzer/instrument_preconditions_locations/s2n_hash_inlined.c

```diff
@@ -0,0 +1,44 @@
+struct evp_md_ctx_st
```

Comment:
❓ What's the license of this file?

Reply:
For `s2n_hash_inlined.c` lines 1-4 are adapted from openssl which has an Apache like licence and lines 5-44 are modified from the AWS s2n library that is also under Apache 2.0 licence. All of `simple.c` is modified from the AWS s2n library.

Reply:
Please add a header and comments so that we know where this is coming from.

Reply:
Done